### PR TITLE
Adds default label to snapshot class with the correct annotation

### PR DESCRIPTION
**Pull request by vbnrh**
Addresses https://issues.redhat.com/browse/RHSTOR-1344
Screenshot from 2020-11-02 14-24-46

Signed-off-by: Vineet Badrinath <vbadrina@redhat.com>

### CODE REVIEW

**Comment by afreen23** [comment]
Why do we need that ? Is this noted in UX designs ?
Please attach the related info.
Thanks!

**Comment by vbnrh** [comment]
> Why do we need that ? Is this noted in UX designs ?
> Please attach the related info.
> Thanks!

Working on it as part of https://issues.redhat.com/browse/RHSTOR-1344. We are implementing the same way as it is for storage classes

File: frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx [hunk 1]
```diff
@@ -19,7 +19,11 @@ const tableColumnClasses = [
   classNames('pf-m-hidden', 'pf-m-visible-on-md'), // Deletion Policy
   Kebab.columnClass,
 ];
-
+const defaultSnapshotClassAnnotation: string = 'snapshot.storage.kubernetes.io/is-default-class';
+export const isDefaultClass = (volumeSnapshotClass: VolumeSnapshotClassKind) => {
```
Comment (cloudbehl) [inline frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx:23]:
```suggestion
export const isDefaultSnapshotClass = (volumeSnapshotClass: VolumeSnapshotClassKind) => {
```

Comment (cloudbehl) [inline frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx:23]:
```suggestion
export const isDefaultClass = (volumeSnapshotClass: VolumeSnapshotClassKind) => 
  getAnnotation(volumeSnapshotClass).?[defaultSnapshotClassAnnotation] === 'true'
```

Comment (vbnrh) [reply frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx]: done

Comment (vbnrh) [reply frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx]: fixed

**Comment by cloudbehl** [comment]
@rawagner please review.

File: frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx [hunk 2]
```diff
@@ -12,14 +12,17 @@ import {
 } from '@console/internal/components/factory';
 import { VolumeSnapshotClassModel } from '@console/internal/models';
 import { getBadgeFromType, BadgeType } from '@console/shared';
+import { getAnnotation } from '@console/kubevirt-plugin/src/selectors/selectors';
```
Comment (rawagner) [inline frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx:15]: do not import from kubevirt plugin

**Review by rawagner — CHANGES_REQUESTED** [review]

File: frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx (hunk 2 above)
Comment (vbnrh) [reply frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx]: sorry for the delayed update. i've fixed this :)

Comment (vbnrh) [reply frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class.tsx]: sorry for delayed update. i've removed kubevirt plugin

**Pull request merged**